internal/handlers/send_email.go

```diff
@@ -13,7 +13,7 @@ import (
 
 func SendEmails(w http.ResponseWriter, r *http.Request) {
 	switch config.Get().MailerModule {
-	case awsModule:
+	case awsModule, "print":
```

Comment:
Tiny nit-pick!  We can probably add this to the constants file as printModule

Reply:
F̴̤̞̹̪̗͎͙̦͚̓̃̌͒̃̊̊͆̎I̶̧̘̻̦̿͌̒́͒̌̌̋͒̇̍͘͝͝͠X̸̢̧̛̪̯͔̹̥͎͇̳̲̭̻̭̔̓̐̑̈̍͊̿̈͂̅͜͝E̸̡̛̳̳̠̫̱̓͂̈́̓̆͆̉̇̈́͛͐͝D̸͇̮̙̝̈́̑͊͌̽̆̏̒́̋̎̉̈́͘͠